src/misc/HtmlSanitizer.ts

```diff
@@ -286,6 +286,12 @@ export class HtmlSanitizer {
 			if (htmlNode.style.position) {
 				htmlNode.style.removeProperty("position")
 			}
+
+			// Don't allow inline images to have a bigger width than the email itself
+			// Otherwise this would lead to weird rendering with very large images and pinch zoom
+			if (htmlNode.tagName === "IMG") {
+				htmlNode.style.maxWidth = "100%"
```

Comment:
does this work in all cases?
Wouldn't it break emails sometimes?

Reply:
Probably fine, just the tests need to be fixed

Reply:
it actually turned out to be a bit problematic with placeholders, that's why we changed the replacement order a bit #e4fafa55ed1edeb262095f53bd2333c4507fbac8